src/os/host-repl.r

```diff
@@ -279,7 +279,7 @@ host-repl: function [
                 ; and needs to be closed.  Invert the symbol.
                 ;
                 unclosed: switch error/arg1 [
-                    "}" ["{"]
+                    "}" ["^{"]
```

Comment:
One of the benefits of using quotes to enclose braces is that you do not need to escape them... any particular reason to add an escaping?